### ISSUE TITLE
[Filter Panel]: QA: Screen reader is announcing irrelavant information when we select any option in the filter panel 

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9295]
### Description

Page Element: 
Screen reader announcement when we select any option from the following filters. 
Product types
Technologies
Business needs
Deployment types

Issue Description:
Screen reader announces the entire content in the filter panel when we select or unselect an option. This happens in each filter category. This Excessive announcement may be annoying for the screen reader user.

Expected:the over verbose announcement should be removed. Check whether inappropriate use of aria-live property is causing this issue.

User Impact:
Any redundant, excessively wordy and extraneous announcement can be distracting for the screen reader users. 

### Component(s) impacted

Filter Panel

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
Closing in favor of #10589
